JournalMonitor/JournalMonitor.cs

```diff
@@ -3818,18 +3818,18 @@ private static Faction getFaction(IDictionary<string, object> data, string type,
                 }
                 else // per-3.3.03 journal
                 {
-                    faction.name = val as string;
+                    faction.name = factionVal as string;
                 }
             }
 
             // Get the faction allegiance
-            if (data.TryGetValue(type + "Allegiance", out val))
+            if (data.TryGetValue(type + "Allegiance", out _))
             {
                 faction.Allegiance = getAllegiance(data, type + "Allegiance");
             }
 
             // Station controlling faction government not discretely available in 'Location' event
-            else if (data.TryGetValue("Factions", out val))
+            else if (data.TryGetValue("Factions", out object val))
```

Comment:
use `stationFactionVal`?

Reply:
Sorry I don't follow.

Reply:
In place of `val` (for a little more specificity). Not important in any case.